hack/cleanup/do-janitor.go

```diff
@@ -14,63 +14,77 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package e2e
+package main
 
 import (
 	"context"
-	"net/http"
+	"log"
 	"os"
-	"strings"
+	"time"
 
 	"github.com/digitalocean/godo"
-	"github.com/pkg/errors"
 )
 
-// CleanDOResources clean any resource leftover from the tests.
-func CleanDOResources(clusterName string) error {
```

Comment:
Why did we remove this function? I'm not sure what exactly it does and did it worked before, but as far as I understand, it would clean leftover resources after the test is done.

This might not be reliable in all cases, for example, if we force push to a branch, Prow might interrupt tests instantly, and that function wouldn't have a chance to finish and clean up resource. But in other cases, it still might be useful, e.g. some failed test or bug.

With Janitor, we would only clean up resources that are 2 days old, every 12 hours, so it might happen that tests leak some resources and that we wait 2 days to remove them, while this function might eventually remove those resources right after tests are finished.

I'm not sure did I understood this correctly, so please correct me if I'm wrong.

Reply:
discussed with @prksu and he suggests to remove from the e2e and use the janitor to deal with that.

we can reduce the time, instead of 2 days we can set to 12 hours or even less.

Reply:
@cpanato Thank you for the explanation! I think reducing the time makes sense. 6-12 hours should be enough, IMO.

Reply:
ok
I also reverted the e2e clean up, that makes sense to keep

Reply:
@xmudrii done, thanks for your review